docs/userguide/components/analysis.md

```diff
@@ -0,0 +1,110 @@
+(analysis_model_userguide)=
+
+# Analysis Models
+
+Analysis models in Earth2Studio provides a set of models that serve as inference sinks
+and pipeline endpoints.
+Unlike {ref}`diagnostic_model_userguide` and {ref}`prognostic_model_userguide` which
+transform and return data, analysis models consume data without returning modified
+tensors or coordinates.
+
+Analysis models are designed for terminal operations in inference pipelines, such as:
+
+- Connections to downstream analysis tools / processes
+
+These models differ fundamentally from diagnostic and prognostic models as they do not
+produce data for further pipeline stages, instead serving as endpoints that process,
+analyze, or consume the input data.
+
+The list of analysis models that are already built into Earth2studio can be found in the
+API documentation {ref}`earth2studio.models.ax`.
+
+## Analysis Interface
+
+The full requirements for a standard analysis model are defined explicitly in the
+`earth2studio/models/ax/base.py`.
+
+```{literalinclude} ../../../earth2studio/models/ax/base.py
+:lines: 25-
+:language: python
+```
+
+:::{note}
+Analysis models do not need to inherit this protocol, this is simply used to define the
+required APIs.
+The key distinguishing feature is that the `__call__` method returns `None`, making
+these models inference sinks rather than data sources or transformers.
```

Comment:
I guess my one complaint about this being a requirement is that in the more interactive/notebook setting, it's nicer to return some analysis "result" directly rather than dumping it to disk and then re-reading it back in later. But I'm not sure if there is a great way to handle arbitrary return types.